nim-mode.el

```diff
@@ -56,6 +56,7 @@
 (require 'nim-compile)
 (require 'nim-suggest)
 (require 'commenter)
+(require 'ob-nim)
```

Comment:
please remove this `(require 'ob-nim)`. Loading this delays to open nim files even users don't need org-mode loading.